src/main/java/com/minecolonies/entity/pathfinding/PathNavigate.java

```diff
@@ -342,13 +379,8 @@ public PathResult moveAwayFromXYZ(BlockPos avoid, double range, double speed)
           null, speed);
     }
 
-    public boolean isUnableToReachDestination()
-    {
-        return pathResult != null && pathResult.failedToReachDestination();
-    }
-
     //We don't use any of these, but they need to be overriden.
-    @Override
+    /*@Override
     public void setAvoidsWater(boolean avoidsWater)
     {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)